packages/components/src/Layout/OneColumn/OneColumn.component.js

```diff
@@ -20,7 +21,13 @@ function OneColumn({ drawers, children, tabs, ...props }) {
 		flexDirection: 'column',
 	};
 	return (
-		<div role="main" id="tc-layout-main" tabIndex="-1" className={container} {...props}>
+		<div
+			role="main"
+			id="tc-layout-main"
+			tabIndex="-1"
+			className={container}
+			{...omit(props, 'getComponents')}
+		>
```

Comment:
This is to reduce console warnings `Warning: React does not recognize the `getComponent` prop on a DOM element.`